patterns/hello_world_pubsub.py

```diff
@@ -106,13 +107,15 @@ def on_failed_publish(self, e: "FailedPublishEvent"):
         print("Connected and Subscribed! Ready to publish\n")
     try:
         while not SHUTDOWN:
-            # Direct publish the message
-            direct_publisher.publish(destination=Topic.of(TOPIC_PREFIX + f"/python/{unique_name}/{msgSeqNum}"), message=outbound_msg)
             msgSeqNum += 1
-            # Modifying the outbond message instead of creating a new one
-            outbound_msg.solace_message.message_set_binary_attachment_string(f'{message_body} --> {msgSeqNum}')
-            outbound_msg.solace_message.set_message_application_message_id(f'sample_id {msgSeqNum}')
-            time.sleep(0.1)
+            # Check https://docs.solace.com/API-Developer-Online-Ref-Documentation/python/source/rst/solace.messaging.config.solace_properties.html for additional message properties
+            additional_properties = {APPLICATION_MESSAGE_ID: f'sample_id {msgSeqNum}'}
+            # Creating a dynamic outbond message 
+            outbound_message = message_builder.build(f'{message_body} --> {msgSeqNum}', additional_message_properties=additional_properties)
```

Comment:
Might be nice to comment that applciation id is override from the value of set in the builder?

Reply:
yeah that's a good idea 👍 will add it now